src/components/modals/columnSettings/ColumnSections.ts

```diff
@@ -95,6 +95,7 @@ class BehaviorSetttingsSection extends AbstractChain<ColumnSettingsHandlerRespon
             case InputType.INLINKS:
             case InputType.OUTLINKS:
             case InputType.METADATA_TAGS:
+            case InputType.CHECKBOX_TYPE:
```

Comment:
All the configuration as metadata info of the checkbox type I think it could be moved to column configuration, the idea of the flag to control the behavior its cool but I think it should be managed on columnConfig